<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-P7VKIKU" name="Test" tests="3" failures="2" timestamp="20 Apr 2018 19:47:34 GMT" time="4.605" errors="0">
  <testcase name="teste01" time="0.001" classname="teste.Testar"/>
  <testcase name="teste02" time="0.0" classname="teste.Testar">
    <failure type="java.lang.AssertionError" message="expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [2]
	at teste.Testar.teste02(Testar.java:48)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- teste02 -->
  <testcase name="teste03" time="0.0" classname="teste.Testar">
    <failure type="java.lang.AssertionError" message="expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [2]
	at teste.Testar.teste03(Testar.java:59)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- teste03 -->
</testsuite> <!-- Test -->
